### PR TITLE
Fix/broken links

### LINKED ISSUE #221: Search broken links, images and videos in repositories
- [x] Developer Guidelines - eoscostarica.io
- [x] DevOps- https://github.com/edenia/openstack-docs
- [x] DevOps - https://github.com/edenia/openstack-docs/blob/master/Kubernetes.md
- [x] DevOps - What is a Container?
- [x] Open Source Projects - https://eos-foundation.eosio.cr - WARNING
- [x] Open Source Projects - https://notarize.eosio.cr/dashboard/notary
- [x] Backend Boilerplate - hasura
- [x] Node installation -  Example
- [x] Block Explorers - https://eospark.com/
- [x] Block Explorers - https://glass.cypherglass.com/map/main/top50
- [x] Block Explorers - https://monitor.telosfoundation.io/
- [x] EOS Podcast - EOS Radio
- [x] EOS Podcast - EOS Dublin
- [x] EOS Podcast - EOS Dapp Stars

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### GH Issue

Resolves #221  

### Steps to test
1.Open link
1.Check content is loaded

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: docs/tutorials/node-installation.md [hunk 1]
```diff
@@ -60,8 +60,6 @@ tail -f stderr.txt
 
 Check if you can access your node using link `http://**YOUR_SERVER**/v1/chain/get_info`
 
-Here is an API [Example](https://latamlink.eosio.cr/v1/chain/get_info)
```
Comment (xavier506) [inline docs/tutorials/node-installation.md:63]: @leisterfrancisco  the correct link here would be : https://lacchain.eosio.cr/v1/chain/get_info

Comment (leisterfrancisco) [reply docs/tutorials/node-installation.md:63]: Solved

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by AngeloCG97 — APPROVED** [review]

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**